app.py

```diff
@@ -50,7 +71,7 @@
     VALID_DIMENSION = {"width": 1920, "height": 1080}
     VALID_EXTENSION = {"jpeg", "jpg", "png", "gif", "bmp", "tiff", "webp"}
 
-  try:
+try:
     MAX_CONTENT_LENGTH_MEGABYTES = int(os.getenv("NACHET_MAX_CONTENT_LENGTH"))
 except (TypeError, ValueError):
     MAX_CONTENT_LENGTH_MEGABYTES = 16
```

Comment:
should probably log a warning here in the log in case  the default value is not what the user was expecting.

"MAX_CONTENT_LENGTH_MEGABYTES value missing or incorrect, setting to 16 MB"

Reply:
Done